src/UIBook.elm

```diff
@@ -540,7 +540,14 @@ init props _ url navKey =
       , actionLogModal = False
       , isMenuOpen = False
       }
-    , maybeRedirect navKey activeChapter
+    , case activeChapter of
+        Just _ ->
+            Cmd.none
+
+        Nothing ->
+            Array.get 0 chapters
+                |> Maybe.map (Nav.replaceUrl navKey << urlFromChapter props.config.urlPreffix)
+                |> Maybe.withDefault Cmd.none
```

Comment:
We need to make sure the user gets redirected to `/` if an invalid path is provided.

```suggestion
                |> Maybe.withDefault (Nav.replaceUrl navKey "/")
```